scalyr_agent/copying_manager.py

```diff
@@ -199,6 +200,26 @@ def __init__(self, configuration, monitors):
         # Keep track of monitors
         self.__monitors = monitors
 
+        # collect monitor-specific extra server-attributes
+        monitor_to_extra_server_attribs = dict(
```

Comment:
This dict is now unused and should be removed (along with the list comprehension).